### PR TITLE
Tw-617/collections main info

### PR DESCRIPTION
**Pull request by Boltman92**

### CODE REVIEW

File: src/screens/collection/collection.styles.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import { createUseStyles } from 'src/styles/create-use-styles';
+import { formatSize } from 'src/styles/format-size';
+
+export const useCollectionStyles = createUseStyles(({ colors, typography }) => ({
+  container: {
+    flex: 1,
+    marginVertical: formatSize(12),
+    marginLeft: formatSize(20)
+  },
+  NFTcontainer: {
```
Comment (svjatoslavtt) [inline src/screens/collection/collection.styles.ts:10]: why do we use NFT and Collection names for one entity?)

File: src/screens/collection/collection.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import { TouchableCollectibleIcon } from '../collectibles-home/collectibles-list/touchable-collectible-icon/touchable-collectible-icon';
+import { useCollectionStyles } from './collection.styles';
+
+export const Collection = () => {
+  const styles = useCollectionStyles();
+
+  const { params } = useRoute<RouteProp<ScreensParamList, ScreensEnum.Collection>>();
+  const collectibles = useCollectiblebyCollectionInfo(params.collectionContract);
+  const tokens = useMemo(() => collectibles.length, [collectibles]);
+  console.log(tokens, 'TOKENS');
```
Comment (svjatoslavtt) [inline src/screens/collection/collection.tsx]: do we need this log?

File: src/screens/collection/collection.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+  const handleChanged = useCallback((info: { viewableItems: ViewToken[]; changed: ViewToken[] }) => {
+    if (isDefined(info.viewableItems) && isDefined(info.viewableItems?.[0].index)) {
+      setInnerScreenIndex(info.viewableItems[0].index);
+    }
+  }, []);
+
+  const renderItem: ListRenderItem<TokenInterface> = ({ item }) => {
+    return (
+      <View style={styles.NFTcontainer}>
```
Comment (svjatoslavtt) [inline src/screens/collection/collection.tsx:33]: maybe better to rename to 'collectible'

File: src/screens/collection/collection.tsx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+              </View>
+            </View>
+          </View>
+        </View>
+      </View>
+    );
+  };
+
+  return (
+    <View style={styles.container}>
```
Comment (svjatoslavtt) [inline src/screens/collection/collection.tsx:62]: and this like 'root'

File: src/screens/collection/collection.styles.ts (hunk 1 above)
Comment (svjatoslavtt) [reply src/screens/collection/collection.styles.ts:10]: OK I maybe confused with the Сollectible, but anyway we use the Collectible name)

**Review by svjatoslavtt — APPROVED** [review]

File: src/apis/objkt/index.ts [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+      return {
+        alias: isDefined(alias) ? alias : undefined,
+        discord: isDefined(discord) ? discord : undefined,
+        github: isDefined(github) ? github : undefined,
+        logo: isDefined(logo) ? logo : undefined,
+        twitter: isDefined(twitter) ? twitter : undefined,
+        tzdomain: isDefined(tzdomain) ? tzdomain : undefined,
+        website: isDefined(website) ? website : undefined
+      };
+    })
```
[the remark this answers is not in the document]
Comment (svjatoslavtt) [reply src/apis/objkt/index.ts:60]: Maybe we haven't empty object but we have some undefined values.

**Review by svjatoslavtt — APPROVED** [review]

File: src/apis/objkt/index.ts (hunk 5 above)
[the remark this answers is not in the document]
Comment (Boltman92) [reply src/apis/objkt/index.ts:60]: bc we sometimes receive object with nullable value

**Pull request closed without merging**